### ISSUE TITLE
sentry_sdk running along side flask throwing Bad Request

### ISSUE THREAD

**Issue by harshad16** [thoth-station/user-api#1769]
**Describe the bug**
sentry_sdk running along side flask throwing Bad Request in user-api v0.35.2
```
Internal error in sentry_sdk
Traceback (most recent call last):
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/wrappers.py\", line 128, in on_json_loading_failed
    return super().on_json_loading_failed(e)
  File \"/opt/app-root/lib64/python3.8/site-packages/werkzeug/wrappers/request.py\", line 611, in on_json_loading_failed
    raise BadRequest(
werkzeug.exceptions.BadRequest: 400 Bad Request: Did not attempt to load JSON data because the request Content-Type was not 'application/json'.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File \"/opt/app-root/lib64/python3.8/site-packages/sentry_sdk/integrations/flask.py\", line 192, in inner
    FlaskRequestExtractor(request).extract_into_event(event)
  File \"/opt/app-root/lib64/python3.8/site-packages/sentry_sdk/integrations/_wsgi_common.py\", line 72, in extract_into_event
    parsed_body = self.parsed_body()
  File \"/opt/app-root/lib64/python3.8/site-packages/sentry_sdk/integrations/_wsgi_common.py\", line 121, in parsed_body
    return self.json()
  File \"/opt/app-root/lib64/python3.8/site-packages/sentry_sdk/integrations/flask.py\", line 172, in json
    return self.request.get_json()
  File \"/opt/app-root/lib64/python3.8/site-packages/werkzeug/wrappers/request.py\", line 571, in get_json
    return self.on_json_loading_failed(None)
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/wrappers.py\", line 133, in on_json_loading_failed
    raise BadRequest() from e
werkzeug.exceptions.BadRequest: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
```

**Additional context**
This is only happening in prod.
[user-api-17-44684-user-api.log](https://github.com/thoth-station/user-api/files/8896805/user-api-17-44684-user-api.log)

**Comment by harshad16** [comment]
similar to: https://github.com/getsentry/sentry-python/issues/1459

**Comment by harshad16** [comment]
This is not critical, it doesnt stops the functionality of user-api

**Comment by harshad16** [comment]
seems like the option is to downgrade the `werkzeug`.
https://www.mail-archive.com/pgadmin-support@lists.postgresql.org/msg04310.html
https://github.com/getsentry/sentry-python/issues/1459#issuecomment-1214131002
https://github.com/pallets/werkzeug/issues/2386#issuecomment-1097372801

**Comment by harshad16** [comment]
Fixed with #1793
Already in production.
Thanks to everyone.
